docs/conf/conf.py

```
@@ -19,6 +19,15 @@
     "sphinxcontrib.mermaid",
 ]
 
+# TODO: These suppressions are global, and while they solve the issue of suppressing 
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)